### DISCUSSION TITLE
Welcome to sql-slow-mirror Discussions!

### DISCUSSION THREAD

**Discussion by lhccong** [lhccong/sql-slow-mirror#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lhccong** [comment]
请大家畅所欲言

**Comment by Shingbb** [comment]
为什么运行不出来

**Reply by lhccong** [reply]
可以检查下环境配置

**Reply by Shingbb** [reply]
我的环境是 JDK17，MySQL8.0

**Reply by lhccong** [reply]
请将环境切换到 JDK8 再试试喔

**Reply by Shingbb** [reply]
JDK8 可以
